scripts/wrappers/disable.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import click
+
+from common.utils import (
+    ensure_started,
+    exit_if_no_permission,
+    is_cluster_locked,
+    print_help,
+    wait_for_ready,
+    xable,
+)
+from status import get_available_addons, get_current_arch, get_status
+
+
+@click.command(context_settings={'ignore_unknown_options': True})
+@click.argument('addons', nargs=-1, required=True)
+def disable(addons):
+    """Disables one or more MicroK8s addons.
+
+    For a list of available addons, run `microk8s status`.
+
+    To see help for individual addons, run:
+
+        microk8s disable ADDON -- --help
+    """
+
+    if print_help(addons):
```

Comment:
Every time we disable an addon we try to print its help?

Reply:
Updated the function name to clarify its purpose, and added a docstring. The function will check to see if a `--help` or `-h` flag was passed in for an addon. If the addon doesn't handle the flag itself, it will print a generic help message and return `True`, meaning that the addon had its help text printed, and should not actually be enabled/disabled. This means that we can migrate the addons as necessary over to handling a `--help` flag.